File: Implementations/PrimeCheck.js
const readline = require("readline-sync"); 

console.log("Enter a number");
let num = Number(readline.question());
for(let i = 2; i < Math.sqrt(num)+1; i++)
{
    if(num % i == 0)
    {
        console.log('%d is not prime: %d*%d', num, i, num/i);
        process.exit()
    }
}
console.log('%d is prime', num);

File: Implementations/PrimeCheck.jl
print("Enter a number\n")
num = parse(UInt64, readline())
for i in 2:floor(sqrt(num))+1
    if num % i == 0
        print(num," is not prime: ", convert(UInt64, i),"*",convert(UInt64, num/i),"\n")
        exit()
    end
end
print(num," is prime\n")
